loan-payment-skip/REPWRITERSPECS/BANNO.LOANPAYMENT.SKIP.V1.POW

```diff
@@ -1106,6 +1111,20 @@ PROCEDURE READCONFIGFILESETTINGS
             VALIDTESTACCOUNT=TRUE
           END
         END
+
+[ get program installation date ]
+       ELSE IF TMPPARAM="PID" AND

```

Comment:
In this block of code should we be setting BNODEBUGMODE to BNODEBUGMODEOFF if we find the param but we are beyond the 90 days?  As written. I think we fall back to the old way of comparing against the last mod date.  I think, to be in sync with how it is described in the CFG, we should set BNODEBUGMODE to BNODEBUGMODEOFF when the param is found but we are beyond the 90 days.  I also noticed there is no initialization on the BNODEBUGMODE variable.  It probably gets set to BNODEBUGMODEOFF automatically but it might not be a bad idea to add the initialization just for clarity.

Reply:
@rrobison44: corrected - BNODEBUGMODE initialized and BNODEBUGMODE remains off if cutoff date (PID + BNOPRINTDEBUGINFODAYS) has been met/passed.